### DISCUSSION TITLE
Have same output in WPF application

### DISCUSSION THREAD

**Discussion by vbaros** [spectreconsole/spectre.console#710] [Q&A] [answered]
I have code which is shared between console app and a WPF app. Is there a way to have same output (with colors, tables...) in an WPF app?

**Comment by phil-scott-78** [comment]
I'm struggling to wrap my head around what you are trying to see. Could you expand upon it?

**Comment by vbaros** [comment]
I'm trying to have the same colourful formatting including tables inside a WPF control.

**Reply by phil-scott-78** [reply]
I suppose you could take a look at something like the [HtmlEncoder](https://github.com/spectreconsole/spectre.console/blob/main/src/Spectre.Console/Internal/Text/Encoding/HtmlEncoder.cs) and take that output and display it in a WebView control, or perhaps write your own for RTF and use the RichTextBox to display it. I'd be tempted to do the latter.

**Comment by patriksvensson** [comment, marked as answer]
Quoting @phil-scott-78 here as the answer:

>I suppose you could take a look at something like the [HtmlEncoder](https://github.com/spectreconsole/spectre.console/blob/main/src/Spectre.Console/Internal/Text/Encoding/HtmlEncoder.cs) and take that output and display it in a WebView control, or perhaps write your own for RTF and use the RichTextBox to display it. I'd be tempted to do the latter.
